### DISCUSSION TITLE
How can I get the list of options for the serverless command?

### DISCUSSION THREAD

**Discussion by marvinirwin** [serverless/serverless#10292] [Q&A]
`npm serverless --help` seems to just start the serverless deploy process.  I know help is a flag because I can see it [https://github.com/serverless/serverless/blob/master/lib/cli/commands-schema/common-options/global.js](here), but when I use it nothing seems to happen

**Comment by pgrzesik** [comment]
I believe the problem in your case is how are you passing help to `serverless` command. Is there a reason as to why you need to run it via `npm`? If so, I think you need to pass flags in this manner: `npm run serverless -- --help`. If you need help about specific command, run `npm run serverless <command> -- --help` instead.

If you're just using `serverless` directly, then `serverless --help` and `serverless <command> --help` will work
